lib/Gitlab/Client.php

```diff
@@ -250,50 +243,34 @@ public function clearHeaders()
      */
     public function setHeaders(array $headers)
     {
-        $this->httpClient->setHeaders($headers);
+        $this->httpClientBuilder->clearHeaders();
+        $this->httpClientBuilder->addHeaders($headers);
 
         return $this;
     }
 
     /**
-     * @param string $name
-     *
-     * @return mixed
-     *
-     * @throws InvalidArgumentException
+     * @param string $api
+     * @return AbstractApi
      */
-    public function getOption($name)
+    public function __get($api)
```

Comment:
FYI, https://github.com/mailgun/mailgun-php/pull/192#discussion_r84417626

We decided to remove it in Mailgun api